cli/src/commands/bookmark/create.rs

```diff
@@ -81,9 +82,26 @@ pub fn cmd_bookmark_create(
     }
 
     let mut tx = workspace_command.start_transaction();
+    let git_settings = tx.settings().git_settings()?;
+    let remotes = tx
+        .repo()
+        .view()
+        .remote_views_matching(&git_settings.auto_track_bookmarks.remote.to_matcher())
+        .map(|(remote, _)| remote.to_owned())
+        .collect_vec();
```

Comment:
nit: we can use `tx.base_repo().clone()` or `workspace_command.repo()` to get around mutable borrow. Extra cloning wouldn't matter, though.